src/components/Badge.astro

```diff
@@ -25,10 +25,12 @@ const inverseColoursMap = {
 }
 
 const map = inverse ? inverseColoursMap : coloursMap;
-const badgeClass =`${map[type]} type-caption-2 px-2 py-0.5 rounded-full border self-center`
+const badgeClass =`${map[type]} type-caption-2 px-2 py-0.5 rounded-full border self-center`;
 
 ---
-{link ?
-  <a class={badgeClass} href={link}>{badgeText}</a> :
-  <span class={badgeClass}>{badgeText}</span>
-}
+<div class="not-prose">
```

Comment:
@AndyClifford this is throwing off your alignment if you look at a heading (e.g. http://localhost:4321/api/bank-account-connection-intents#create-bank-account-connection-intent)

Suggested change:
```
const badgeClass =`${map[type]} type-caption-2 px-2 py-0.5 rounded-full border`;

---
<span class="not-prose flex items-center">
  {link ?
    <a class={badgeClass + ' underline'} href={link}>{badgeText}</a> :
    <span class={badgeClass}>{badgeText}</span>
  }
</span>
```

Reply:
Do you mean the experimental badge drops down onto the next line?

Reply:
<img width="480" alt="image" src="https://github.com/centrapay/centrapay-docs/assets/64108933/122e19ed-e08e-44e6-bc23-d7ed49792482">

Reply:
@AndyClifford it's not aligned on the vertical axis. I've changed classes in the suggestion too :)